packages/amplify-graphql-relational-transformer/src/schema.ts

```diff
@@ -369,7 +390,7 @@ function makeModelXFilterInputObject(
         kind: 'Name',
         value: 'and',
       },
-      type: makeListType(makeNamedType(name)) as any,
+      type: makeListType(makeNamedType(name)) as unknown as NamedTypeNode,
```

Comment:
Though the attributes match with `NamedTypeNode`, I think it should be `ListTypeNode` here looking at the code.

Reply:
ListTypeNode is incompatible with the fields array, but since I reverted the use of TSAny to any in the other file, will just revert this line back